src/stan/interface_callbacks/writer/stream_writer_typedefs.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef STAN_INTERFACE_CALLBACKS_WRITER_STREAM_WRITER_TYPEDEFS_HPP
+#define STAN_INTERFACE_CALLBACKS_WRITER_STREAM_WRITER_TYPEDEFS_HPP
+
+#include <stan/interface_callbacks/writer/stream_writer.hpp>
+#include <sstream>
+#include <iostream>
+
+namespace stan {
+  namespace interface_callbacks {
+    namespace writer {
+      // An implementation of stream_writer
+      // using stringstream for unit tests
+      typedef stream_writer<std::stringstream> sstream_writer;
```

Comment:
We shouldn't need this typedef when the templating
is replaced with the simpler OO pattern.

> On Oct 9, 2015, at 10:34 AM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/interface_callbacks/writer/stream_writer_typedefs.hpp:
> 
> > @@ -0,0 +1,18 @@
> > +#ifndef STAN_INTERFACE_CALLBACKS_WRITER_STREAM_WRITER_TYPEDEFS_HPP
> > +#define STAN_INTERFACE_CALLBACKS_WRITER_STREAM_WRITER_TYPEDEFS_HPP
> > +
> > +#include <stan/interface_callbacks/writer/stream_writer.hpp>
> > +#include <sstream>
> > +#include <iostream>
> > +
> > +namespace stan {
> > +  namespace interface_callbacks {
> > +    namespace writer {
> > +      // An implementation of stream_writer
> > +      // using stringstream for unit tests
> > +      typedef stream_writer<std::stringstream> sstream_writer;
> 
> Any reason we don't put this typedef inside stream_writer.hpp?
> 
> —
> Reply to this email directly or view it on GitHub.
>